runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -1880,6 +1904,23 @@ J9::Z::TreeEvaluator::inlineIntrinsicIndexOf(TR::Node * node, TR::CodeGenerator
    regDeps->addPostCondition(resultVector, TR::RealRegister::AssignAny);
    regDeps->addPostCondition(valueVector, TR::RealRegister::AssignAny);
 
+   // Offset to be added to array object pointer to get to the data elements
+   int32_t offsetToDataElements = TR::Compiler->om.contiguousArrayHeaderSizeInBytes();
+#ifdef J9VM_GC_SPARSE_HEAP_ALLOCATION
+   if (TR::Compiler->om.isOffHeapAllocationEnabled())
+      {
+      // Load first data element address
+      generateRXInstruction(cg,
```

Comment:
Similar comment as I had previously, TR::Register `array` is not clobberable register.